src/sentry/api/endpoints/project_key_details.py

```diff
@@ -123,7 +123,7 @@ def put(self, request, project, key_id):
                 request=request,
                 organization=project.organization,
                 target_object=key.id,
-                event=AuditLogEntryEvent.PROJECTKEY_EDIT,
+                event=AuditLogEntryEvent.PROJECTKEY_EDIT_RATE_LIMIT,
```

Comment:
Has the rate limit necessarily been edited? It looks like there are multiple pathways through this method that would result in the rate limit not being modified. Perhaps there the new rate limit audit log should be in addition to the project update audit?

Reply:
Can you explain what you mean by the rate limit not being edited? 
![Screen Shot 2019-03-08 at 11 58 53 AM](https://user-images.githubusercontent.com/25037561/54052569-91987400-4199-11e9-9789-ccabf733a54f.png)

Reply:
The PUT method supports renaming a key. A simple rename operation would not modify the rate limit and the audit logs would be incorrect.

Reply:
I'm still not sure that this change is correct. Renaming or deactivating a key will now show up as a rate-limit change. Should the event type decision be moved up to where the changes are done?

If a project key has its ratelimit and name changed in a single request (something the UI doesn't do but the API supports) would we want the rate limit audit log or a general edit audit log or two logs?